aidies-project/client/src/components/UsersList.js

```diff
@@ -8,9 +8,10 @@ function UsersList (props) {
 
     useEffect(() => {
         async function fetchData() {
-            const response = await fetch(apiUrl + '/users/');
+            const response = await fetch(apiUrl + '/session/');
```

Comment:
I know that this was left over from the starter, but just a reminder note that you can remove this component if it's not helpful for your project!